ci/Jenkinsfiles/build.groovy

```diff
@@ -218,6 +249,56 @@ pipeline {
         }
       }
     }
+    stage('Build Web UI Docker Image') {
+      steps {
+        setGitHubBuildStatus('docker/webui', 'Build Web UI Docker Image', 'PENDING')
+        container('maven') {
+          echo """
+          -------------------------
+          Build Web UI Docker image
+          -------------------------
+          """
+          dir('docker/webui') {
+            script {
+              TMP_NAMESPACE = "${APP_NAME}-${BRANCH_NAME.toLowerCase()}-tmp"
+
+              // deploy
```

Comment:
maybe explain with more details what this deploy is for? (since it's a bit of a hack to do a full deployment to grab the deployed files)